try {
    console.log("Start");
    errorCode;
    console.log("Finsih");
} catch (error) {
    console.log("There is an error, so this code wont't be running");
}

/*Output:
Start
There is an error, so this code won't be running
*/
